src/node_crypto.cc

```diff
@@ -1608,6 +1611,51 @@ void SSLWrap<Base>::RequestOCSP(
 }
 
 
+template <class Base>
+void SSLWrap<Base>::GetEphemeralKeyInfo(
+    const v8::FunctionCallbackInfo<v8::Value>& args) {
+  Base* w = Unwrap<Base>(args.Holder());
+  Environment* env = Environment::GetCurrent(args);
+  HandleScope handle_scope(env->isolate());
```

Comment:
Let me confirm this.  Isn't the HandleScope necessary because Local Object of info passed to ReturnValue and the ReturnValue takes care of the life of info? That means there are no local handles to be disposed within GetEphemeralKeyInfo.